app/javascript/components/rooms/RoomSettings.jsx

```diff
@@ -32,7 +43,26 @@ export default function RoomSettings() {
           <Row>
             <Col className="border-end border-2">
               <h6 className="text-primary">Room Name</h6>
+
+              <Button
+                variant="primary-light"
+                // Generates the access code: a string of six characters
+                onClick={() => setAccessCode(Math.random().toString(36).slice(2, 8))}
```

Comment:
Will change this to do a Mutation that generates a (new) Access Code via backend onClick.